pkg/prometheus/ha/server.go

```diff
@@ -259,11 +237,7 @@ func (s *Server) notifyReshard(ctx context.Context, desc *ring.IngesterDesc) err
 	}
 	defer cli.Close()
 
-	backoff := util.NewBackoff(context.Background(), util.BackoffConfig{
-		MinBackoff: time.Second,
-		MaxBackoff: 2 * time.Minute,
-		MaxRetries: 10,
-	})
+	backoff := util.NewBackoff(context.Background(), backoffConfig)
```

Comment:
What's the reason for passing `context.Background()` here instead of the original `ctx`?

Reply:
I was blindly copying things :) Nice catch, I'll fix it.